example/storybook/stories/TrackTile/AdvancedTrackerEditor.stories.tsx

```diff
@@ -43,6 +43,7 @@ storiesOf('AdvancedTrackerEditor', module)
 
     Object.assign(ontology, nutrition);
     let color = '#33C317';
+    let unitCode = "[U'arb]";
```

Comment:
During testing I noticed this resulted in fractional values for procedures if not defined because of unit conversions